styleguide/derek/global-components/basement/basement.ejs

```diff
@@ -1,15 +1,24 @@
 <!--[basement]-->
 <div class="bg-dark-gray">
   <div class="basement-container">
-    <div class="col-sm-4">
+    <div class="col-sm-9">
       <div class="region-select">
         <span class="uppercase white basement-region-select">Change your region:</span>
         <a class="basement-flag-link" data-toggle="modal" id="region" data-target="#regionSelect">
           <div class="basement-flag-icon basement-flag-uk"></div>
         </a>
       </div>
+      <div class="basement-currency col-lg-pull-1">
```

Comment:
If you're gonna do this, probably could just add: `@include make-lg-column-pull(1);` to the definition of basement-currency.